internal/action/actions.go

```diff
@@ -745,7 +805,7 @@ func (h *BufPane) Autocomplete() bool {
 	}
 	r := h.Cursor.RuneUnder(h.Cursor.X)
 	prev := h.Cursor.RuneUnder(h.Cursor.X - 1)
-	if !util.IsAutocomplete(prev) || !util.IsNonAlphaNumeric(r) {
+	if !util.IsAutocomplete(prev) || !util.IsNonWordChar(r) {
```

Comment:
I would have used `util.IsWordChar(r)` in the first place, but hey...my style is sometimes confusing too.
Furthermore update the comment to not autocomplete in the moment the cursor is on a word character?

Reply:
Hmm, yeah, I haven't noticed this... Yeah, `IsWordChar()` here IMO looks definitely cleaner.

Hmm... actually we can just remove it: `IsAutocomplete()` internally already calls `!IsNonWordChar()`, no need to call it again here.

Reply:
> Hmm, any chance you read [...]

Yep indeed, sorry for confusion.

> How about changing the comment to:
> 
> ```go
> // don't autocomplete if cursor is on a word character (within a word)
> ```

Or just:

```go
// don't autocomplete if cursor is within a word
```

?

Or maybe closer to the original version (more clear)?

```go
// don't autocomplete if cursor is middle of a word
```

Reply:
> > Hmm, any chance you read [...]
> 
> Yep indeed, sorry for confusion.

I did exactly the same fault. Sorry, then it can stay!

Reply:
Still, changing `!util.IsNonWordChar()` to `util.IsWordChar()` is a good idea, right?

Reply:
I updated the PR and tested the autocomplete functionality. Works like it did before for me.

Regarding the comment I went with
```go
// don't autocomplete if cursor is within a word
```
because auto-completion is also disabled when you are on the first or last character, not just when you are on middle ones. Hope that's fine.